### ISSUE TITLE
Enum ValidValues sinceVersion is always 0

### ISSUE THREAD

**Issue by billsegall** [real-logic/simple-binary-encoding#414]
For the golang backend I want to provide a sinceVersion()/inActingVersion() for each ValidValue in an enum.

The spec of June 16 2016 suggests in section 4.5.6.2 that sinceVersion is an element attribute of a ValidValue.

When looking at ValidValue tokens it seems that token.version() is always equal to zero for these regardless of how it's set.

I've been testing with the following
```:
--- a/sbe-tool/src/test/resources/basic-types-schema.xml
+++ b/sbe-tool/src/test/resources/basic-types-schema.xml
@@ -13,8 +13,8 @@
         </composite>
         <type name="EDT" description="char array" length="20" primitiveType="char" semanticType="char"/>
         <enum name="ENUM" description="enum as uint8" encodingType="uint8" semanticType="int">
-            <validValue name="Value1">1</validValue>
-            <validValue name="Value10">10</validValue>
+            <validValue name="Value1" sinceVersion="1">1</validValue>
+            <validValue name="Value10" sinceVersion="2">10</validValue>
```

**Comment by mjpt777** [comment]
The `validValue` and `choice` elements carried forward the `sinceVersion` and `description` of the field rather than what was set on these elements. I've updated the parser to correct this.

**Comment by billsegall** [comment]
Thanks @mjpt777 . I can confirm the fix of #c6822f1 works for me.
